File: 6-2/2/2.py
import random

hundrednum = []
for i in range(100) :
    hundrednum.append(random.randint(1, 1000))

for i in range(len(hundrednum)) :    
    print(hundrednum[i], end= " ")

def findMax(ls) :
    maxnum = 0
    for i in range(len(ls)) :
        if ls[i] > maxnum :
            maxnum = ls[i]
    return maxnum

def findMin(ls) :
    minnum = 1001
    for j in range(len(ls)) :
        if ls[j] < minnum :
            minnum = ls[j]
    return minnum

print()
print('max value:', findMax(hundrednum))
print()
print('min value:', findMin(hundrednum))

File: 8-1/2/2.c
#include <stdio.h>

int main(){
    int n1, n2, n3;
    scanf("%d %d %d", &n1, &n2, &n3);
    int min, max;
    if( n1 <= n2 && n1 <= n3){
        min = n1;
    }
    else if ( n2 <= n1 && n2 <= n3){
        min = n2;
    }
    else{
        min = n3;
    }

    if(n1 >= n2 && n1 >= n3){
        max = n1;
    }
    else if( n2 >= n1 && n2 >= n3){
        max = n2;
    }
    else{
        max = n3;
    }
    printf("min: %d \nmax: %d", min, max);
}